rosidl_generator_py/rosidl_generator_py/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 try:
     from .generate_py_impl import generate_py
+    assert generate_py
```

Comment:
That's another good reason to just use `flake8`... Oh well. Thanks for looking into it @ernestmc.